src/makeCalculator.js

```diff
@@ -38,6 +38,42 @@
  */
 function makeCalculator() {
   // write code here
+  const calculator = {
+    result: 0,
+
+    operate(callback, number) {
+      this[callback.name](number);
+
+      return this;
+    },
+
+    add(number) {
+      this.result += number;
+    },
+
+    subtract(number) {
+      this.result -= number;
+    },
+
+    multiply(number) {
+      this.result *= number;
+    },
+
+    divide(number) {
+      if (number === 0) {
+        return 'error';
```

Comment:
Try to create a more descriptive message
```suggestion
        return 'Can not divide by 0, try another number';
```